pyvista/core/dataset.py

```diff
@@ -1315,16 +1345,16 @@ def flip_z(self, point=None, transform_all_input_vectors=False, inplace=False):
 
         Parameters
         ----------
-        point : list, optional
+        point : sequence[float], optional
```

Comment:
numpydoc edits earlier in the file used point: FloatVector, here it's sequence[float]

Reply:
```suggestion
        point : FloatVector, optional
```